client/src/components/AssetTable/AssetRow/AssetRow.js

```diff
@@ -105,47 +109,50 @@ function AssetRow(props) {
     setSelected(false);
   }, [selectList, selected, setSelectList, asset.asset_tag]); //calls on changes to select list to work with the Clear Selection Button
 
-  var categoryLabel = cats.map((cat) => cat.category_id === asset.category ? cat.catName : "");
+  var categoryLabel = cats.map((cat) =>
+    cat.category_id === asset.category ? cat.catName : ""
+  );
+
+  // Refreshes the available state when the checked_out or operational states change
+  useEffect(() => {
+    setAvailable(asset.checked_out || !asset.operational);
+  }, [asset.checked_out, asset.operational]);
 
   return (
     <tr className={selected ? "table-primary" : null}>
       <AccessControl allowedRank={Ranks.OPERATOR}>
         <td>
           {/*The button below creates a shopping cart icon next to the asset that changes on a successful add.*/}
           <Button
-            variant={
-              asset.checked_out ? "danger" : selected ? "success" : "secondary"
-            }
+            variant={available ? "danger" : selected ? "success" : "secondary"}
             onClick={handleSelect}
-            disabled={asset.checked_out ? true : false}
+            disabled={available ? true : false}
           >
             <img
               alt={
-                asset.checked_out
+                available
                   ? "Unavailable"
                   : selected
-                    ? "Added to Cart"
-                    : "Add to Cart"
-              }
-              src={
-                asset.checked_out ? crossedOut : selected ? checkMark : cartIcon
+                  ? "Added to Cart"
+                  : "Add to Cart"
               }
+              src={available ? crossedOut : selected ? checkMark : cartIcon}
               width="25"
               height="25"
             />
           </Button>
         </td>
       </AccessControl>
-      <td><Link to={`/asset/${asset.asset_tag}`} state={{ asset, categoryLabel }}>{asset.asset_tag}</Link></td>
+      <td>
+        <Link to={`/asset/${asset.asset_tag}`} state={{ asset, categoryLabel }}>
+          {asset.asset_tag}
+        </Link>
+      </td>
       <td>{asset.name}</td>
       <td>{asset.description}</td>
       <td>{formattedDate}</td>
-      <td>
-        {
-          categoryLabel
-        }
-      </td>
-      <td>{asset.checked_out ? "No" : "Yes"}</td>
+      <td>{categoryLabel}</td>
+      <td>{available ? "No" : "Yes"}</td>
```

Comment:
This might look better as the icons used in the Single Asset Pages

Reply:
Good idea, I can implement that as above and beyond the DOD.